tests/testlogin.js

```diff
@@ -0,0 +1,91 @@
+//const { assert } = require("console");
+//const { default: $ } = require("webdriverio/build/commands/browser/$");
+
+describe ('TestLogin', () => {
+
+   
+    it('Login  with correct username and correct password', async () => {
```

Comment:
Consider using the naming convention of structuring your test case's names with a **should** based sentence. In your case, something like: "**Should** login with...". Same with every other test.